src/InstagramClone.Api/DataTranferaobjects/UserCreateDTO.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace InstagramClone.Api
+{
+    public class UserCreateDTO
+    {
+        public Guid Id { get; set; } //PK
+        public string Email { get; set; }
+        public string Password { get; set; }
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public string Avatar { get; set; }
+    }
+}
```

Comment:
The class is okay, but is there a reason why the folder is called `DataTranferaobjects`? 

Can you suggest a better or simpler and shorter name for the folder?

Reply:
Ok.I will simply use DTOs. I am adding the S because I want to believe  we will use other Dtos for register and login

Reply:
https://www.cosmiclearn.com/dotnet/aspnet_webapi_httppost.php

This is the link where I saw the return CreatedAtRoute(...)

Reply:
Now I return the createdAtRoute methed and  and return Ok response .
now it works.

![image](https://user-images.githubusercontent.com/59874300/178146146-bcb2469b-afcd-4413-b04d-38f306ff3367.png)